modules/contributor-guide/partials/proc_extending-styles.adoc

```diff
@@ -0,0 +1,52 @@
+:_module-type: PROCEDURE
+
+[id="proc_extending_styles_{context}"]
+= Extending styles in the vale-at-red-hat repository
+
+The vale-at-red-hat repository is an open source project maintained and developed by volunteers. If you want to help developing `vale-red-hat`, feel free to open a PR.
```

Comment:
The backticks make this look like it might be a specific object or folder. How about:
```suggestion
The vale-at-red-hat repository is an open-source project maintained and developed by volunteers. If you want to contribute to the Vale at Red Hat project, feel free to open a PR.
```